### DISCUSSION TITLE
Webhooks Issue with USER content-type using Zapier

### DISCUSSION THREAD

**Discussion by Ryuk-hash** [strapi/strapi#6600] [Legacy Topics]
Hi,

User.js file location: 
`extensions > users-permissions > controllers > User.js`
So I've been having trouble working with webhooks. User is a content-type just like any other. But there seems to be some issue when I try to create a new user, or updated any user. The controller does its job, the issue lies with Zapier, that is, it doesn't register the PUT/GET requests for the User (e.g. when I create a new user, Zapier doesn't receive a request). For some reason, the DELETE request seems to be working fine. Kindly help me figure this out, as I'm in need of implementing a scenario that: When I'm on a webpage viewing a list of my team members, and a new member gets added to the team, the webpage should render the change from the backend instead of reloading the webpage. I figured webhooks were the go-to for this, but I'm welcome to any other suggestions! Thank you! :)

Requests not getting received by Zapier:
Create a new user
`POST /content-manager/explorer/plugins::users-permissions.user 200`

Update a user
`PUT /content-manager/explorer/plugins::users-permissions.user/149 200`

![image](https://user-images.githubusercontent.com/50495162/84142216-05e35680-aa72-11ea-8f60-3694b96d3fbf.png)

Request getting received by Zapier:
`DELETE /content-manager/explorer/deleteAll/plugins::users-permissions.user?0=184 200`

![image](https://user-images.githubusercontent.com/50495162/84142078-cddc1380-aa71-11ea-9911-fc409e7a05cb.png)

**Comment by Ryuk-hash** [comment]
Confirming that this is a prevailing issue already: https://github.com/strapi/strapi/issues/5324
What other alternative could be to my use-case? Use life-cycle callbacks? If yes, then how? Please help out as I'm new to Strapi! @derrickmehaffy

**Reply by derrickmehaffy** [reply]
Lifecycle callbacks would be an option

**Reply by Ryuk-hash** [reply]
Unhelpful.
